config/gvmd.service.in

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Greenbone Vulnerability Manager (gvmd)
```

Comment:
Maybe we could even use:

```suggestion
Description=Greenbone Vulnerability Manager daemon (gvmd)
```

Any opinions?

Reply:
It's getting a bit long, and other places like INSTALL use "Greenbone Vulnerablity Manager".  Also, "Manager" is a noun already, so "Green Vulnerability Management daemon" might be better grammar.

Reply:
But if `Greenbone Vulnerability Management daemon` is used this could cause confusion because "gsad" is also a "Greenbone Vulnerability Management daemon".

Reply:
That sounds wrong.  GSA is for "Greenbone Security Assistant".

Reply:
But if gsad has "daemon", then good if gvmd has the same.

Reply:
I guess we need to decide between one of these variants:

- Greenbone Vulnerability Manager daemon (gvmd)
- Greenbone Vulnerability Manager (gvmd)
- Greenbone Vulnerability Manager
- Greenbone Vulnerability Manager daemon

because "Greenbone Vulnerability Management daemon" sounds wrong for me and will cause the confusion with GVM where e.g. ospd-openvas and gsad are parts of.

Reply:
For me it is `Greenbone Vulnerability Management daemon (gvmd)` (https://gvm-tools.readthedocs.io/en/latest/glossary.html)

Reply:
Thanks for clarification, i guess then the whole gvmd repos needs various updates:

https://github.com/greenbone/gvmd#greenbone-vulnerability-manager

Reply:
So we came to the following conclusion which was also reflected in https://github.com/greenbone/gvm-tools/pull/279:

```suggestion
Description=Greenbone Vulnerability Manager daemon (gvmd)
```